packages/core/src/bus/Command.ts

```diff
@@ -0,0 +1,11 @@
+import { CommandDispatcher } from './CommandDispatcher';
+import { BusInstruction } from './BusInstruction';
+
+export abstract class Command<T, R> extends BusInstruction<T> {
+  public readonly expectReply: boolean = true;
+  public readonly ttl: number = 10000;
```

Comment:
You can omit type declarations here since they will be inferred
```suggestion
  public readonly expectReply = true;
  public readonly ttl = 10000;
```
```suggestion
  public readonly expectReply: boolean = true;
  public readonly ttl: number = 10000;
```